charts/nexus3/values.yaml

```diff
@@ -26,6 +26,10 @@ serviceAccount:
 
 deployment: true
 
+# if deployment = false, statefulset will be used. If these replicas are set > 1, then HA options will be used.
+# Be aware that you need to fulfill a few prerequisites for a HA deployment. Please check sonatype documentation!
+replicasStatefulset: 1
```

Comment:
```suggestion
replicas: 1
```

Reply:
Do you really think we should just call it replicas? I've left deployment with replicas = 1 hardcoded, because the HA way could just work with statefulset due to the persistent volume - it could work with tmpdir in deployment as well, but f.e. we are using the options to use persistent volumes to keep f.e. logs. Multiple deployment pods won't work with just one ReadWriteOnce PV(C)...

Reply:
The `Deployment` would keep it's hard coded replica count, the chart is focused on the `StatefulSet` implementation and `replicas` is the idiomatic value for my charts.